src/OptimisticTokenVotingPlugin.sol

```diff
@@ -184,13 +185,18 @@ contract OptimisticTokenVotingPlugin is
     }
 
     /// @inheritdoc IOptimisticTokenVoting
+
     function totalVotingPower(uint256 _timestamp) public view returns (uint256) {
         return votingToken.getPastTotalSupply(_timestamp);
+        // Q(danielw): we should probably need to remove votes of `taikoL1` which holds a lot of TAIKO as bond deposits.
+        // return votingToken.getPastTotalSupply(_timestamp) - votingToken.getPastVotes(address(taikoL1), _timestamp);
     }
 
     /// @inheritdoc IOptimisticTokenVoting
     function bridgedVotingPower(uint256 _timestamp) public view returns (uint256) {
+         // Q(danielw): I think we should use `taikoERC20Vault`, not `taikoBridge`.
```

Comment:
My bad. I had been using that value from the list of addresses you provided, assuming that it was like an alias. Should replacing the name of the variable globally be enough, or is there any use for the `taikoBridge` related to the DAO?

Reply:
The bridge contract is not supposed to hold any TAIKO token, so you can ignore its token balance. the bridge contract will only be used here to aggregate L2 votes later.